mmv1/third_party/terraform/website/google.erb.tmpl

```diff
@@ -36,6 +36,11 @@
         <li>
           <a href="/docs/providers/google/guides/version_3_upgrade.html">3.0.0 Upgrade Guide</a>
         </li>
+<!-- Commented out until 4.0.0 release
```

Comment:
We can release it early, no harm in that- just means folks will have more details about deprecations

AWS had one kicking around a year in advance of a breaking change, at one point!

Reply:
Great I'll remove the comment